modules/claims_api/app/sidekiq/claims_api/poa_updater.rb

```diff
@@ -4,7 +4,7 @@
 
 module ClaimsApi
   class PoaUpdater < ClaimsApi::ServiceBase
-    def perform(power_of_attorney_id) # rubocop:disable Metrics/MethodLength
+    def perform(power_of_attorney_id, form_number) # rubocop:disable Metrics/MethodLength
```

Comment:
@stiehlrod Based on the failed tests I suspect that the spots where the `PoaUpdater` jobgets called need this variable added